### DISCUSSION TITLE
Move selected to organisation

### DISCUSSION THREAD

**Discussion by palicepta** [dani-garcia/vaultwarden#4189] [Q&A] [answered]
Hello All,

I got the error when tried to move the passwords to another organization.
Somebody had the same error?

bitwarden-vaultwarden-1  | [2023-12-20 15:08:35.114][request][INFO] PUT /api/ciphers/share
bitwarden-vaultwarden-1  | [2023-12-20 15:08:35.117][vaultwarden::api::core::ciphers][ERROR] You must select at least one cipher.
bitwarden-vaultwarden-1  | [2023-12-20 15:08:35.117][response][INFO] (put_cipher_share_selected) PUT /api/ciphers/share => 400 Bad Request

Thank you in advance for any help!

<img width="472" alt="Screenshot 2023-12-20 at 5 23 21 PM" src="https://github.com/dani-garcia/vaultwarden/assets/112872766/8890c046-a8a4-4442-bbdf-931fd34b9479">

**Comment by BlackDex** [comment]
The cipher you want to move is probably not owned by you but a different organization.

Also, moving multiple ciphers between collections is not supported by the clients. They just do not send those ciphers to the server.

**Comment by palicepta** [comment]
Hello [BlackDex](https://github.com/BlackDex)

Does it mean that we cannot move some passwords between organizations?

**Comment by palicepta** [comment, marked as answer]
Hello all,

I did it by export and import function
Thanks all for your help
